src/index.html

```diff
@@ -6,9 +6,41 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a class ="logo" href="#">
+        <img class="logo__img"
+        src="images/logo.png"
+        alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li
+          class="nav__item"><a class="nav__link is-active" href="#">APPLE</a></li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a 
              class="nav__link is-active"
              href="#"
            >APPLE</a>
          </li>
```
or
```suggestion
          <li class="nav__item">
            <a 
              class="nav__link is-active"
              href="#"
            >
              APPLE
            </a>
          </li>
```

Reply:
follow code style, move attributes to new line if you have >=2 attrs
move every attribute to new line with +2 indentation